Update fhir bundle

stefpiatek opened the pull request:
- Broadly updating to changes in bundle, though I think we'll probably need to develop a JSON schema and validate against it, have tests for example inputs and outputs to ensure they are the same or both
- Refactored initial values to be shared between tests and app for during development. Can then have initial values as blank in the production application
- Discussion and suggested changes to example bundle is in #14, but included the file here so that there is a reference to it already in the repository

Closes #24

jhughes982 reviewed (changes requested):
The code look good!

However, when testing the form locally there seem to be a few data discrepancies (could just be my lack of understanding though!). For instance, during the 'variant' form section the `gene information` field is empty and it looks like this is because there is no corresponding `geneInformation` field under the variant parent field in `FormDefaults.ts` (although there is one under the `result` parent field). I can explain this in person if I'm not making sense.

I also noticed that the final bundle contains data that is not presented in the form (e.g. `clinicalConclusion`).

I think before we merge this one we should make sure the data and final bundle are all looking good.

stefpiatek commented on src/components/reports/ReportForm.tsx:
```diff
@@ -1,8 +1,7 @@
-import { useContext, useState, useRef } from "react";
+import { useContext, useRef, useState } from "react";
```
I guess must be prettier reordering them alphabetically?

stefpiatek commented:
Agreed on the bundle. In #26 I found that using the .shape in the Yum schema meant that the type was `any`, and addressed issues there - I think probably worth getting this PR merged in and then update #26, and use that as the source of truth for the bundle following through perfectly because then we're using the type safety
